### ISSUE TITLE
`is_private_collector` vs `is_private_collection` is confusing...

### ISSUE THREAD

**Issue by dchiller** [DDMAL/CantusDB#1693]
Suggestion: keep the current name for `is_private_collection` (identifies that an institution is an individual/private collector regardless of having a RISM siglum) since there the field name is straightforward. For `is_private_collector`, at least change the public-facing name to something more descriptive (`no_rism_siglum`?), but perhaps just change the field.

**Comment by dchiller** [comment]
Besides its use in the constraint that institutions should generally have RISM sigla, I'm not totally sure whether we need this field either.

**Comment by ahankinson** [comment]
> Besides its use in the constraint that institutions should generally have RISM sigla, I'm not totally sure whether we need this field either.

That's its whole purpose -- there needs to be a way to conditionally require the RISM siglum be entered, and the way to do that is to have this field, where either it, or the siglum, but not both, are filled. The idea is that, if it doesn't have a RISM siglum, there is no "public" registration of the details of the collection. (the assumption is that if you have an entry in RISM with a siglum, then you are making a public statement that you have some books, and that there is a way to contact you to find out more. So it's not "private" as in "a private person" it's private as in "the details of the collection are not available to the public."

The "is_private_collection" field was added afterwards, to meet the requirement that individual holders of MSS are identifiable, whether or not they have a RISM siglum assigned. It was the best I could come up with.

If either of them were to change, I would suggest the latter (`is_private_collection`) -- Possibly "individual collector" or something like that. It's easier because it's not really used anywhere, whereas the setup for the `is_private_collector` touches a bunch of code.

**Comment by dchiller** [comment]
Yeah, I understand the history/rationale. My point is only that it doesn't currently mean anything in addition to an institution without RISM siglum. In any case, this is the way it is now and doesn't seem worth revisiting in the absence of a compelling reason; maybe someday users will be allowed to add institutions (this could help us force them to do the right thing) or there will be a secret third type of institution to add to our Venn diagram and we'll do more with this checkbox.

In any case, the current display means that Debra/Anna don't know what to do with these checkboxes and I have to look at the help text each time because I'm afraid I'll mix up "collector" vs "collection", which is really the problem. 

What about we change the display for `is_private_collector` to just "Is Private" (keeps "private" per the meaning @ahankinson notes above -- similar to the "private notes" field -- and avoids substantial code changes) and the `is_private_collection` field to `collection_type` which can default to "Institutional" but be changed to "Individual" or "Private"? 

I'll run by Debra too to see if that distinction seems clearer for her continued use.

**Comment by ahankinson** [comment]
>  and the is_private_collection field to collection_type which can default to "Institutional" but be changed to "Individual" or "Private"?

That sounds like a good direction. What do you think of something like "organization" or "individual" to distinguish between corporate vs. personal collections?

I would avoid "institutional" or "private" because these are now "loaded" terms (they have different meanings in the context of Cantus now). 

"Type" can also be a generic term -- you can have a collection "type" of prints, (as in, the only items in the collection are prints)  for example, that have a different meaning than the word "type" here. Perhaps something like "Ownership: Organization | Individual" ?

**Comment by dchiller** [comment]
> What do you think of something like "organization" or "individual" to distinguish between corporate vs. personal collections?

This is great, imho! I realized to have an "Institution" of type "Institutional" made no sense, but "Organization" is perfect.

> Perhaps something like "Ownership: Organization | Individual" ?

Also makes a lot of sense!

**Comment by dchiller** [comment]
Ok, there exists general buy-in on "Private" boolean and "Ownership: Organization | Individual" so we'll go ahead with that...
